docs/rules/comma-spacing.md

```diff
@@ -123,7 +123,7 @@ If your project will not be following a consistent comma-spacing pattern, turn t
 
 ## Further Reading
 
-* [Javascript](http://javascript.crockford.com/code.html)
+* [JavaScript](http://javascript.crockford.com/code.html)
```

Comment:
Note that this change must be made in the main eslint/eslint repo because that’s the source of our docs. Making the change here will only update the live site and will disappear in the next release.

Reply:
Thanks for the heads up @nzakas :+1:

https://github.com/eslint/eslint/pull/13730